lib/btf.c

```diff
@@ -296,11 +293,23 @@ int open_bpf_map(psabpf_context_t *psabpf_ctx, const char *name, psabpf_btf_t *b
     if (errno_val != NO_ERROR)
         return errno_val;
 
-    /* Find entry in BTF for our map */
+    /* Find BTF type IDs for our map */
+    md->key_type_id = 0;
+    md->value_type_id = 0;
     if (btf != NULL && btf->btf != NULL) {
-        md->btf_type_id = psabtf_get_map_type_id_by_name(btf->btf, name);
-        if (md->btf_type_id == 0)
+        uint32_t btf_type_id = psabtf_get_map_type_id_by_name(btf->btf, name);
+        if (btf_type_id == 0)
             fprintf(stderr, "can't get BTF info for %s\n", name);
+
+        if (md->map_key_type_id == 0)
```

Comment:
I didn't notice it before but using `if` without parenthesis `{ }` is not considered as a clean code. Please always use parenthesis for `if` and `else` from now on.